tools/topology/sof-imx8qxp-cs42888.m4

```diff
@@ -52,7 +52,7 @@ dnl     period, priority, core, time_domain)
 DAI_ADD(sof/pipe-dai-playback.m4,
 	1, ESAI, 0, ESAI0-Codec,
 	PIPELINE_SOURCE_1, 2, s24le,
-	1000, 0, 0)
+	1000, 0, 0, SCHEDULE_TIME_DOMAIN_DMA)
```

Comment:
Commit description and change don't match on this one.

Reply:
sorry didn't get this... you want me to add there that this is especially time domain "dma" not "timer"

Reply:
You set it to DMA here as requested but the commit message still says "timer" if I read it right.

Reply:
My bad, it says time, not timer. Please ignore this.

Reply:
commit message says just time domain, not is it "timer" or "dma", but I will clarify it :)